setup.py

```diff
@@ -28,8 +28,16 @@
     packages=['mdbenchmark'],
     package_data={'mdbenchmark': ['templates/*']},
     install_requires=[
-        'numpy>=1.8', 'mdsynthesis', 'click', 'jinja2', 'pandas', 'matplotlib',
-        'python-Levenshtein', 'xdg<2',
+        'numpy>=1.8',
+        'mdsynthesis',
+        'click',
+        'jinja2',
+        'pandas',
+        'matplotlib',
+        'python-Levenshtein',
+        'xdg<2',
     ],
     entry_points={'console_scripts': ['mdbenchmark=mdbenchmark.cli:cli']},
+    setup_requires=['pytest-runner'],
+    tests_require=['pytest', 'mock', 'pytest-cov'],
```

Comment:
please remove coverage for the tests. This is only useful for CI runs and occasionally on a dev machine. Adding coverage usually makes tests slower to run.

Reply:
is the mock really needed?

Reply:
We need `mock` to test the responses from system packages like `socket.gethostname`. This way we can test it on all machines. If we wouldn't do that, then we'd need to write tests specifically for Travis-CI and these would fail on any other machines.
See for example: https://github.com/bio-phys/MDBenchmark/pull/3/files#diff-4c190294c14d18b75f1b5a206c73944cR32

Reply:
Oh wait. I just realized that we're anyway using the `monkeypatch` fixture from `pytest`. It's possible that we can actually get rid of the `mock` dependency then.